build-tools/scripts/RunTests.targets

```diff
@@ -11,7 +11,8 @@
     <_NUnit>$(_Runtime) packages\NUnit.ConsoleRunner.3.7.0\tools\nunit3-console.exe</_NUnit>
     <_Test Condition=" '$(TEST)' != '' ">--test=&quot;$(TEST)&quot;</_Test>
     <_XABuild>$(_TopDir)\bin\$(Configuration)\bin\xabuild</_XABuild>
-    <_XABuildProperties>/p:Configuration=$(Configuration) /p:XAIntegratedTests=$(XAIntegratedTests)</_XABuildProperties>
+    <_XABuildDiag Condition=" '$(V)' != '' ">/v:diag </_XABuildDiag>
+    <_XABuildProperties>$(_XABuildDiag)/p:Configuration=$(Configuration) /p:XAIntegratedTests=$(XAIntegratedTests)</_XABuildProperties>
```

Comment:
This shouldn't hold up this PR, but does it need a space?

    $(_XABuildDiag)/p:Configuration=$(Configuration)

Should be:

    $(_XABuildDiag) /p:Configuration=$(Configuration)

Reply:
Look at the `$(_XABuildDiag)` definition: it has a *trailing* space. Therefore, no space is needed. here.